lib/openstates/models/bill.rb

```diff
@@ -32,7 +32,7 @@ def api_method
       end
 
       def id_key
-        :bill_id
+        :id
```

Comment:
This change will only let one of the two endpoints work if I understand it correctly. This may need some more thought before we merge this change.